core/debug/src/debug.c

```diff
@@ -376,10 +376,12 @@ static void debug_fault_hard(void)
 
 void debug_init(void)
 {
+#if !defined(ARCH_MPU_ARMv8M)
```

Comment:
According to the Cortex M33 user guide, there is no DISDEFWBUF bit in SCnSCB->ACTLR (this bit is reserved):
![image](https://user-images.githubusercontent.com/14541274/29667532-6c4cf0ea-88e5-11e7-9201-2911dd9fcfb0.png)

Reply:
Without this, I wonder if we still get good debug information from the MMFAR or SFAR.